static/src/stylesheets/layout/_header.scss

```diff
@@ -179,3 +147,648 @@ $c-guardian-services-action: #ffffff;
+
+    &::-webkit-details-marker {
+        display: none;
+    }
+
+    .main-menu__icon--chevron:before {
+        border-color: $news-support-2;
+    }
+}
+
+.main-menu__icon--chevron {
+    //optical alignment
+    margin-top: -.2em;
+
+    &:before {
+        content: '';
+        display: inline-block;
+        width: 8px;
+        height: 8px;
+        transform: rotate(45deg);
+        border-width: 2px;
+        border-style: solid;
+        border-top: 0;
+        border-left: 0;
+    }
+
+    details[open] & {
+        margin-top: .1em;
+
+        &:before {
+            transform: rotate(-135deg);
+        }
+    }
+}
+
+.main-menu__icon {
+    position: absolute;
+    left: $gs-gutter / 2;
+    text-align: center;
+    width: $navigation-horizontal-padding - $gs-gutter/2;
+
+    &__svg {
+        fill: $news-support-2;
+        margin-top: -.05em;
+        width: 1em;
+        height: 1em;
+    }
+}
+
+.main-navigation__secondary {
+    color: #ffffff;
+    background: darken($guardian-brand-dark, 4%);
+
+    .main-menu__icon__svg {
+        margin-top: -.24em;
+        width: .8em;
+        height: .8em;
+    }
+}
+
+.navigation-border,
+[data-link-name='nav2 : the guardian app'] {
+    // Creates a seamless line when section has dark background
+    margin-top: -1px;
+    position: relative;
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: $navigation-horizontal-padding;
+        right: 0;
+        height: 1px;
+        background: darken($guardian-brand-dark, 4%);
+    }
+}
+
+[data-link-name='nav2 : the guardian app'],
+[data-link-name='nav2 : facebook'] {
+    margin-top: $gs-baseline * 2;
+}
+
+.navigation-group {
+    // Unset ul from _lists.scss
+    margin-left: 0;
+    // Unset ul from user agent stylesheet
+    margin-top: 0;
+    // Unset ul from user agent stylesheet
+    margin-bottom: 0;
+}
+
+.navigation-group--contrast {
+    color: $guardian-brand-dark;
+    background-color: $news-main-2;
+    padding-bottom: $gs-baseline;
+
+    .main-menu__icon__svg {
+        fill: $guardian-brand;
+    }
+
+    .personalisation__icon:before {
+        border-color: $guardian-brand;
+    }
+}
+
+.navigation-group__item {
+    // Override inherited ul from user agent stylesheet
+    list-style: none;
+    position: relative;
+    font-size: 1em;
+
+    details[open] &:last-child {
+        padding-bottom: $gs-baseline;
+    }
+
+    a {
+        display: block;
+        padding-top: $gs-baseline / 1.5;
+        padding-right: $gs-gutter;
+        padding-bottom: $gs-baseline / 1.5;
+        padding-left: $navigation-horizontal-padding;
+
+        // Override a from user agent stylesheet
+        color: inherit;
+
+        // TODO: Share with large nav?
+        &:focus,
+        &:hover {
+            text-decoration: none;
+            // Unset a:focus from user agent stylesheet
+            outline: 0;
+        }
+    }
+}
+
+.secondary-navigation {
+    font-size: .9em;
```

Comment:
@zeftilldeath might remember why this choice was made better than me. I think its because in the blue part of the subnav has a proportionally smaller font. I believe it was done like this because the font sizes changed a lot over the past few months, and its much easier just to change it in one place rather than many. Open to changing it, `18px` is the right value in its current size